eox_tenant/admin.py

```diff
@@ -54,5 +56,15 @@ def course_org_filter(self, microsite):
         except Exception as error:
             return str(error)
 
+    def ednx_signal(self, microsite):
+        """
+        Read only method to see if the site has activated the usage of signals
+        """
+        # pylint: disable=broad-except
+        try:
+            return microsite.values.get('EDNX_USE_SIGNAL', "EMPTY")
+        except Exception as error:
+            return str(error)
```

Comment:
We can catch a NameError exception, since get returns a None by default and values returns an empty list, so the only known error that could be raised is an NameError, if microsite is not defined for some reason.

Reply:
We have a broad in all methods of this page because we want to make sure that nothing prevents the page from rendering